pac4j-oauth/src/main/java/org/pac4j/oauth/exception/OAuthCredentialsException.java

```diff
@@ -2,15 +2,15 @@
 
 import java.util.*;
 
-import org.pac4j.core.exception.CredentialsException;
+import org.pac4j.core.exception.TechnicalException;
 
 /**
  * This class represents an exception occurring during OAuth credentials retrieval.
  * 
  * @author Jerome Leleu
  * @since 1.3.0
  */
-public class OAuthCredentialsException extends CredentialsException {
+public class OAuthCredentialsException extends TechnicalException {
```

Comment:
In fact, the internal `retrieveCredentials` method plays the same role as the `Authenticator`  and will soon be replaced by it (https://github.com/pac4j/pac4j/issues/622). So I think it makes sense.